tests/schemas/test_openapi.py

```diff
@@ -830,13 +830,9 @@ def test_serializer_validators(self):
         assert properties['regex']['pattern'] == r'[ABC]12{3}'
         assert properties['regex']['description'] == 'must have an A, B, or C followed by 1222'
 
-        assert properties['decimal1']['type'] == 'number'
-        assert properties['decimal1']['multipleOf'] == .01
-        assert properties['decimal1']['maximum'] == 10000
-        assert properties['decimal1']['minimum'] == -10000
+        assert properties['decimal1'] == {'type': 'string', 'format': 'decimal'}
```

Comment:
I opted for this style of validation to ensure there are no other properties being erroneously added to the spec, and breaking validation.